tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+# Additional helper functions specific to Anthropic
+def extract_anthropic_tool_calls(response: Any) -> List[Dict[str, Any]]:
+    """Extract tool calls from Anthropic response format with proper type checking"""
+    tool_calls = []
+
+    # Type check for Anthropic Message response
+    if not hasattr(response, "content") or not response.content:
+        return tool_calls
+
+    for content in response.content:
+        if hasattr(content, "type") and content.type == "tool_use":
+            if hasattr(content, "name") and hasattr(content, "input"):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)